test/rules/Username-attribute.test.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const loadRule = require('../utils/load-rule');
+const ContextBuilder = require('../utils/contextBuilder');
+const RequestBuilder = require('../utils/requestBuilder');
+
+const ruleName = 'add-country';
```

Comment:
This is executing the wrong rule. To execute your rule, please change this to the rule filename that you settle on for the file above.

Reply:
Fixed